packages/components/src/components/form/Input/Input.tsx

```diff
@@ -198,7 +198,11 @@ const Input = ({
             </InputWrapper>
 
             {bottomText && (
-                <BottomText inputState={inputState} isDisabled={isDisabled}>
+                <BottomText
```

Comment:
I would assume that shall be just injectable component and you can send it through props and not just hardcoded `BottomText`

Reply:
I think here we should have something like `bottomTextRendered()` which is a function that can render custom bottom text and we can pass `inputState` as a parameter of the `bottomTextRendered` to inject it